conf/prod.routes

```diff
@@ -1,4 +1,5 @@
-->      /                                                  health.Routes
+->      /                                                                                  health.Routes
 GET     /guides/safety-and-security-import-declarations-end-to-end-service-guide           controllers.Default.redirect(to = "/guides/safety-and-security-import-declarations-end-to-end-service-guide/")
 GET     /guides/safety-and-security-import-declarations-end-to-end-service-guide/          controllers.Assets.at(path="/public", file="index.html")
 GET     /guides/safety-and-security-import-declarations-end-to-end-service-guide/*file     controllers.Assets.at(path="/public", file)
+GET     /*file                                                                             controllers.Assets.at(path="/public", file)
```

Comment:
Hi @preetampotdar. Although this works locally, you _might_ run into problems when you build in Jenkins and then deploy to the environments. I tried the [same solution for the customs-declarations-end-to-end-service-guide](https://github.com/hmrc/customs-declarations-end-to-end-service-guide/releases/tag/v0.239.1), and it didn't work for that. I would suggest building a release from your branch first, and then deploying to QA to see what effect it has.